### PR TITLE
admin control part 1

### PR DESCRIPTION
**Pull request by huynhduc96**
screenshot from 2018-07-30 09-58-17

### CODE REVIEW

File: app/assets/stylesheets/custom.scss [hunk 1]
```diff
@@ -14,3 +14,9 @@
 .margin_li_user {
   margin: 10px;
 }
+.tab_fix_width{
+  width: 180px
+}
+.center-lg{
```
Comment (tuquyet2709) [inline app/assets/stylesheets/custom.scss]: cách dòng ra

File: app/views/admin/users/index.html.erb [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+        <div class="course_info_title">Leader waiting</div>
+        <div class="course_info_text"><%= @leader_no_active.count %></div>
+      </div>
+
+    </div>
+
+    <!-- Course Tabs -->
+    <div class="course_tabs_container">
+      <div class="tabs d-flex flex-row align-items-center justify-content-start">
+        <div class="tab active tab_fix_width">Leader</div>
```
Comment (tuquyet2709) [inline app/views/admin/users/index.html.erb]: i18n all file please

File: app/views/admin/users/index.html.erb [hunk 3]
```diff
@@ -0,0 +1,58 @@
+<% provide :title, t(".title") %>
+
+<!-- Course -->
+<div class="col-lg-8 center-lg">
+
+  <div class="course_container">
+    <div class="course_title">User Controler</div>
```
Comment (hngda01) [inline app/views/admin/users/index.html.erb]: i18n

File: app/views/admin/users/index.html.erb [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+
+      <!-- Course Info Item -->
+      <div class="course_info_item">
+        <div class="course_info_title">Leader</div>
+        <div class="course_info_text"><%= @leader_activated.count %></div>
+      </div>
+
+      <!-- Course Info Item -->
+      <div class="course_info_item">
+        <div class="course_info_title">Member</div>
```
Comment (hngda01) [inline app/views/admin/users/index.html.erb]: i18n

File: app/views/admin/users/index.html.erb [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+        <div class="course_info_text"><%= @member.count %></div>
+      </div>
+
+      <!-- Course Info Item -->
+      <div class="course_info_item">
+        <div class="course_info_title">Leader waiting</div>
```
Comment (hngda01) [inline app/views/admin/users/index.html.erb]: i18n

File: app/views/admin/users/index.html.erb [hunk 6]
```diff
@@ -0,0 +1,58 @@
...
+        <div class="course_info_text"><%= @leader_no_active.count %></div>
+      </div>
+
+    </div>
+
+    <!-- Course Tabs -->
+    <div class="course_tabs_container">
+      <div class="tabs d-flex flex-row align-items-center justify-content-start">
+        <div class="tab active tab_fix_width">Leader</div>
+        <div class="tab tab_fix_width">Member</div>
```
Comment (hngda01) [inline app/views/admin/users/index.html.erb]: i18n

File: app/views/admin/users/index.html.erb [hunk 7]
```diff
@@ -0,0 +1,58 @@
...
+        <div class="tab tab_fix_width">Leader waiting</div>
```
Comment (hngda01) [inline app/views/admin/users/index.html.erb]: i18n

**Comment by tuquyet2709** [comment]
ok

**Comment by hngda01** [comment]
ok

**Pull request merged**